angular-client/src/pages/charging-page/components/active-status/active-status.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+import Storage from 'src/services/storage.service';
+import Theme from 'src/services/theme.service';
+import { IdentifierDataType } from 'src/utils/enumerations/identifier-data-type';
+import { floatPipe } from 'src/utils/pipes.utils';
+
+@Component({
+  selector: 'active-status',
+  templateUrl: './active-status.component.html',
+  styleUrls: ['./active-status.component.css']
+})
+export default class ActiveStatus {
+  isActive: boolean = false;
+  currentSeconds: number = 0;
+  totalSeconds: number = Number(sessionStorage.getItem('active total seconds')) || 0;
```

Comment:
can you make all these tags be in this format: 'active-total-seconds', can you also make another ticket to add a reset button to each of these timers/ investigate their behavior to make sure they work correctly. This seems to be working, even if you navigate away, but I'm pretty sus of this, it'll probably break/ be a bit unreliable (e.g. if you open it in two different browsers)

Reply:
true, it only allows for the total value to not go away on reload, otherwise not actually real. I'll make a ticket.